### DISCUSSION TITLE
How to translate "String Literals"?

### DISCUSSION THREAD

**Discussion by iamsuperwen** [python/python-docs-zh-tw#402] [General]
As I working on #41, I encountered "7.1.1. Formatted String Literals".
Which would be the better translation of it, 字串實字 or 字串常數?

**Comment by Nienzu** [comment]
I think you can simply translate it into "字串".
You can take the [link](https://github.com/python/python-docs-zh-tw/blob/c340b24f46a1610e5831ed5fc3a8e836b2f1b790/tutorial/introduction.po#L237) for reference.

**Comment by yichung279** [comment]
c/cpp 社群似乎翻譯為「字串常數」
https://www.ptt.cc/bbs/C_and_CPP/M.1463157729.A.28B.html

**Comment by adrianliaw** [comment]
I think the word "literal" is still important enough to include it in the translation, there's a difference between using and without using the vocabulary "literal", e.g. `"some string"` is a string literal, `"some" + "string"` is not a string literal but is a string.  For "formatted string literals" here, the syntax can only be used when it's a string literal.

I think "字串常數" is a plausible translation as its property of being a constant is [documented inside Python's language reference](https://docs.python.org/3/reference/lexical_analysis.html#literals), I've also seen people translating it to "字串文本", "字串常值", "字串字面值" etc.   All the translations sound unnatural to me anyways, but I personally prefer "字串文本" just because "字串常數" can be confusing to those who come from programming languages that have constants.

Anyhow, for each translated document that has the word "string literal" or "bytes literal" or something similar in it, we should always quote the original English word for the first appearance in that document, see [tutorial/introduction.po](https://github.com/python/python-docs-zh-tw/blob/3.7/tutorial/introduction.po#L152) for an example.

**Comment by iamsuperwen** [comment]
Thanks for the explanation.
> For each translated document that has the word "string literal" or "bytes literal" or something similar in it, we should always quote the original English word for the first appearance in that document.

In sum, let's take "字串文本" as "string literal".

**Comment by iamsuperwen** [comment]
Maybe just leave this open for discussion?
